shared/oval/ensure_gpgcheck_globally_activated.xml

```diff
@@ -9,18 +9,30 @@
       of an RPM package's signature always occurs prior to its
       installation.</description>
       <reference source="MED" ref_id="20130807" ref_url="test_attestation" />
-      <!-- rhel7: <reference source="SDW" ref_id="20131223" ref_url="test_attestation" /> -->
     </metadata>
-    <criteria>
-      <criterion comment="check value of gpgcheck in /etc/yum.conf" test_ref="test_ensure_gpgcheck_globally_activated" />
+    <criteria operator="OR">
+      <criteria operator="AND">
+        <extend_definition comment="Fedora installed" definition_ref="installed_OS_is_fedora" />
+        <criterion comment="check value of gpgcheck in /etc/dnf/dnf.conf" test_ref="test_dnf_ensure_gpgcheck_globally_activated" />
+      </criteria>
+      <criterion comment="check value of gpgcheck in /etc/yum.conf" test_ref="test_yum_ensure_gpgcheck_globally_activated" />
     </criteria>
   </definition>
-  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check value of gpgcheck in /etc/yum.conf" id="test_ensure_gpgcheck_globally_activated" version="1">
-    <ind:object object_ref="object_ensure_gpgcheck_globally_activated" />
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check value of gpgcheck in /etc/yum.conf" id="test_yum_ensure_gpgcheck_globally_activated" version="1">
+    <ind:object object_ref="object_yum_ensure_gpgcheck_globally_activated" />
   </ind:textfilecontent54_test>
-  <ind:textfilecontent54_object id="object_ensure_gpgcheck_globally_activated" comment="gpgcheck set in /etc/yum.conf" version="1">
+  <ind:textfilecontent54_object id="object_yum_ensure_gpgcheck_globally_activated" comment="gpgcheck set in /etc/yum.conf" version="1">
```

Comment:
@mpreisler this object is supposed to be object_yum_... are it is for checking `/etc/yum.conf`. The OVAL checks `/etc/dnf/dnf.conf` if the OS is Fedora; otherwise, it checks `/etc/yum.conf`.